griptape/artifacts/json_artifact.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import annotations
+
+import json
+from typing import Any, Union
+
+from attrs import define, field
+
+from griptape.artifacts import BaseArtifact
+
+Json = Union[dict[str, "Json"], list["Json"], str, int, float, bool, None]
+
+
+@define
+class JsonArtifact(BaseArtifact):
+    _obj: Any = field(metadata={"serializable": True}, alias="obj")
```

Comment:
What does this field provide?

Reply:
this is just the original object being converted. different than `value`.